src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub use zenoh::config as zenoh_config;
 use zenoh::internal::runtime::Runtime as ZRuntime;
 use zenoh::{bytes::ZBytes, pubsub::Subscriber, qos::Priority, Session};
 
+const WILDCARD_ENTITY_TYPE: u32 = 0x0000_FFFF;
```

Comment:
we should probably make these constants public in up-rust ...

Reply:
I'm also thinking about this...
Instead of exposing `WILDCARD_ENTITY_TYPE`, perhaps we can have a function to get the entity type directly?

Reply:
Can you create a corresponding issue in up-rust?

Reply:
Sure